### PR TITLE
End to end testing using playwright

### PR DESCRIPTION
**Pull request by svdwyer**

### CODE REVIEW

File: tests/popup-page.js
Comment (pheyvaer) [inline tests/popup-page.js:1]: Documentation is missing from the functions.

**Review by pheyvaer — CHANGES_REQUESTED** [review]

File: .prettierrc
[the remark this answers is not in the document]
Comment (svdwyer) [reply .prettierrc:1]: correct

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,12 @@ dist
...
\ No newline at end of file
+*.zip
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
```
[the remark this answers is not in the document]
Comment (svdwyer) [reply .gitignore:13]: they shouldn't be. I think I may have run the playwright installation script twice

File: tests/main.spec.js [hunk 2]
```diff
@@ -0,0 +1,417 @@
+import {test, expect} from './fixtures';
+
+// const WEBID_A = 'https://pod.playground.solidlab.be/a/profile/card#me';
+// const WEBID_B = 'https://pod.playground.solidlab.be/b/profile/card#me';
+// const WEBID_C = 'https://pod.playground.solidlab.be/c/profile/card#me';
```
[the remark this answers is not in the document]
Comment (svdwyer) [reply tests/main.spec.js:5]: these comments actually should be removed. the WEBIDs are not used

File: tests/main.spec.js [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+  await identities.getByRole('button', {
+    name: 'C Profile',
+  }).click();
+  await expect(page.getByRole('heading', {name: 'C Profile'}),).toBeVisible();
+});
+
+test('manage profiles popup display empty list if no profiles exist', async ({popupPage}) => {
+  const settingsPage = await popupPage.openSettings();
+
+  await expect(settingsPage).toHaveTitle(/Manage SOLID profiles/);
```
Comment (pheyvaer) [inline tests/main.spec.js:160]: Should be "Solid".

File: tests/main.spec.js [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+test('add button opens add-new-profile page', async ({ page, popupPage, context}) => {
+  await popupPage.openPopup();
+
+  const pagePromise = context.waitForEvent('page');
+
+  await page.getByRole('button', {name: 'Add'}).click();
+
+  const popup = await pagePromise;
+  await popup.waitForLoadState();
+  await expect(popup).toHaveTitle(/Add a new SOLID profile/);
```
Comment (pheyvaer) [inline tests/main.spec.js:27]: Should be "Solid".

**Review by pheyvaer — CHANGES_REQUESTED** [review]

**Review by pheyvaer — APPROVED** [review]

**Pull request merged**